podcasts/EpisodeBookmarksTabsView.swift

```diff
@@ -45,7 +87,7 @@ private extension View {
 
 struct EpisodeBookmarksTabsView_Previews: PreviewProvider {
```

Comment:
Nit: the preview struct should be renamed too.

Reply:
Also the file name. I wish Xcode's Refactor Rename function would work for once. 🤦‍♂️